model-optimizer/requirements.txt

```diff
@@ -2,7 +2,7 @@ tensorflow>=1.15.2,<2.0; python_version < "3.8"
 tensorflow>=2.0; python_version >= "3.8"
 mxnet>=1.0.0,<=1.5.1
 networkx>=1.11
-numpy>=1.14.0,<1.19.0
```

Comment:
Instead of bumping the numpy version we should just fix the MO code. This error comes from the unit test invocation. So probably the bug is in the test.

Any way, we should not force user to migrate to significantly newer version of tools just because of unit-test bug.

@pavel-esir , could you take a look at this issue and prepare a fix in the MO/unit-test?

Reply:
No problem, feel free to close this PR when the fix will be ready.